apps/jobs/views.py

```diff
@@ -228,3 +232,65 @@ def leaderboard(request, challenge_phase_split_id):
     paginator, result_page = paginated_queryset(distinct_sorted_leaderboard_data, request)
     response_data = result_page
     return paginator.get_paginated_response(response_data)
+
+
+@throttle_classes([UserRateThrottle])
+@api_view(['GET'])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_remaining_submissions(request, challenge_phase_pk, challenge_pk):
+    challenge_model = get_model_object(Challenge)
+    challenge = challenge_model(challenge_pk)
+
+    challenge_phase_model = get_model_object(ChallengePhase)
+    challenge_phase = challenge_phase_model(challenge_phase_pk)
+
+    participant_team_pk = get_participant_team_id_of_user_for_a_challenge(
+        request.user, challenge_pk)
+
+    try:
+        ParticipantTeam.objects.get(pk=participant_team_pk)
+    except ParticipantTeam.DoesNotExist:
+        response_data = {'error': 'You haven\'t participated in the challenge'}
+        return Response(response_data, status=status.HTTP_403_FORBIDDEN)
+
+    max_submission_per_day = challenge_phase.max_submissions_per_day
+
+    max_submission = challenge_phase.max_submissions
+
+    submissions_done_today_count = Submission.objects.filter(
+        challenge_phase__challenge=challenge_pk,
+        participant_team=participant_team_pk,
+        challenge_phase=challenge_phase_pk,
+        submitted_at__gte=timezone.now().date()).count()
+
+    failed_submissions_count = Submission.objects.filter(
+        challenge_phase=challenge_phase_pk,
+        participant_team=participant_team_pk,
+        challenge_phase__challenge=challenge_pk,
+        status=Submission.FAILED,
+        submitted_at__gte=timezone.now().date()).count()
+
+    if ((submissions_done_today_count - failed_submissions_count) >= max_submission_per_day
+            or (max_submission_per_day == 0)):
+        date_time_now = timezone.now()
+        date_time_tomorrow = date_time_now.date() + datetime.timedelta(1)
+        utc = pytz.UTC
```

Comment:
@taranjeet I didn't get what exactly @deshraj wanted to say? I mean how we can get time from settings module as we are using `postgres` as database. Please refer here: https://docs.djangoproject.com/en/1.10/ref/settings/#time-zone